apps/src/applab/sanitizeHtml.js

```diff
@@ -170,6 +170,10 @@ export default function sanitizeHtml(unsafe, warn, persistingHtml, rejectExistin
           warnings.push('element id is already in use: ' + attribs.id);
           delete attribs.id;
         }
+        if (attribs.type === 'password') {
```

Comment:
Do we also want to check if `tagName === 'input'` here?  To let students set 'password' as a class, for example.